src/data/markdown/translated-guides/en/03 Using k6 browser/03 Metrics.md

```diff
@@ -36,32 +36,23 @@ The following snippet is an example:
 
 <Blockquote mod="note" title="">
 
-In future releases, the `browser_*` metrics will be removed in favor of the web vitals. As Google also recommends measuring the 75th percentile for each web vital metric, there will still be future tweaks to improve the summary output.
+As Google also recommends measuring the 75th percentile for each web vital metric, there will still be future tweaks to improve the summary output. When the browser module receives the metric from the [Google's Web Vital Library](https://github.com/GoogleChrome/web-vitals), it counts how many times it sees that metric. 
```

Comment:
This statement `When the browser module receives the metric from the [Google's Web Vital Library](https://github.com/GoogleChrome/web-vitals), it counts how many times it sees that metric.` -> We can remove this statement for now. Once we have a blog post or guide on how someone could use Prometheus and Grafana to retrieve the ratings (whether it's good, needs improvement, or poor), then we can link it in the future.

Reply:
Resolved in 0df9299287c77c12bc98881a64f6694d52a23af7.